stl/inc/chrono

```diff
@@ -2215,6 +2215,36 @@ namespace chrono {
         return hours{_Ret};
     }
 
+    // [time.zone.timezone]
+
+    // TRANSITION: work in progress
+    class time_zone {
+    public:
+        time_zone(std::string_view _Name) : _Name(_Name) {}

```

Comment:
We conventionally avoid shadowing between parameters and data members. (Shadowing is generally confusing, especially local-local shadowing, but parameter/data member shadowing is also worth avoiding.) Our usual convention is to add a trailing underscore to the parameter: `_Name_`. Occurs below.